ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/vars/RedHat.yml

```diff
@@ -42,14 +46,11 @@ Build_Tool_Packages:
   - zip
 
 Additional_Build_Tools_RHEL7:
-  - bison
-  - curl-devel
```

Comment:
This is required for compiling git the role.

Reply:
On my RHEL7 system `curl-devel` doesn't appear to be in the repos but `libcurl-dev` (as added to the earlier section) is and is what's installed on the adoptopenjdk RHEL7 systems so are you sure on that package name?

```
[sxa@sxat470p vars]$ yum search curl
Loaded plugins: downloadkvmonly-background, ibm-check-lotus-updates, ibm-check-upgrade, ibm-check-xorg-updates, ibm-repository, langpacks, post-transaction-actions, refresh-packagekit, versionlock
================================================================================================ N/S matched: curl ================================================================================================
libcurl-devel.i686 : Files needed for building applications with libcurl
libcurl-devel.x86_64 : Files needed for building applications with libcurl
perl-WWW-Curl.x86_64 : Perl extension interface for libcurl
python-pycurl.x86_64 : A Python interface to libcurl
curl.x86_64 : A utility for getting files from remote servers (FTP, HTTP, and others)
libcurl.i686 : A library for getting files from web servers
libcurl.x86_64 : A library for getting files from web servers

  Name and summary matches only, use "search all" for everything.
[sxa@sxat470p vars]$ 
```

Reply:
Saying that the GIT_source role attempts to check for it (https://github.com/AdoptOpenJDK/openjdk-infrastructure/commit/1566b820c3b61b7c206ba8791faa1c9949163bee Added by @bblondin) but it seems to not cause a problem if it's not there ...

Reply:
Agreed it should be removed.  Its not on my rh6 or 7 (x86) systems either.  

It was added as part of this Nagios pr, but seems unrelated based on comments.  https://github.com/AdoptOpenJDK/openjdk-infrastructure/pull/255/files.